### DISCUSSION TITLE
Hi all! I tried to connect EdgexGateway and CloudGateway but I got an error this is shown below as! I hope everybody gives me some advice. Thank you so much!!!!!!!!!!!:pray:

### DISCUSSION THREAD

**Discussion by SavantEVO** [edgexfoundry/community#47] [Show and tell]
<img width="972" alt="image" src="https://user-images.githubusercontent.com/109493900/217517282-9f81c355-9ce8-4156-9c86-839736fd9f35.png">
<img width="706" alt="image" src="https://user-images.githubusercontent.com/109493900/217517360-46dfd76b-cb15-40be-9c96-d4133d964884.png">

**Comment by bnevis-i** [comment]
Everyone is waiting on port 54329, which is opened by edgex-security-bootstrapper detects when the persistence layer of EdgeX has booted.  If you look at the above output, edgex-security-bootstrapper is having trouble connecting to edgex-core-consul to verify that consul is up and running.

Please see https://docs.edgexfoundry.org/2.3/design/adr/security/0009-Secure-Bootstrapping/ for more details about this design.  Note that the snap implementation of EdgeX doesn't need this as systemd can deterministically start services.

**Reply by bnevis-i** [reply]
I am unable.  If you need support, continue to use the message boards, or for higher levels of support, I suggest looking at commercial EdgeX implementations, such as those offered by IOTech <https://www.iotechsys.com/products/>
